distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -666,6 +667,14 @@ type Table
             on_problems.attach_problems_before problems <|
                 this.updated_context_and_columns new_ctx new_columns
 
+    ## Parsing values is not supported in database tables, the table has to be
+       materialized first with `to_dataframe`.
+    parse_values : Data_Formatter -> (Nothing | [Column_Type_Selection]) -> Problem_Behavior -> Table
+    parse_values parser=Data_Formatter column_types=Nothing on_problems=Report_Warning =
+        _ = [parser, column_types, on_problems]
```

Comment:
that looks odd?

Reply:
Well, the arguments are unused because this function is just a 'placeholder' to have the API aligned with In-Memory table and report a helpful error.

Moreover, I don't think I can replace the arguments itself with underscores, because this would break named calls - see example above. 

So to avoid unused argument warnings, I need _some_ way to mark them as unused, and this seems like the right way. I'm happy for alternative suggestions, I dont' see these.

Example why we cannot just use underscores:

```
from Standard.Base import all

type A

A.foo x=1 y=2 z=3 = x+y+z

type B

B.foo _=1 _=2 _=3 = 0

main =
    IO.println A.foo
    IO.println B.foo

    IO.println (A.foo y=10)
    IO.println (B.foo y=10)
```

leads to

```
6
0
14
Execution finished with an error: Type error: expected a function, but got 0 (Integer).
        at <enso> argument<1>(args.enso:16:17-26)
        at <enso> args.main(args.enso:16:5-27)
```

The issue is - if the argument names are inconsistent, the named application will not work in the underscore variant. But the whole purpose of this placeholder was to keep the API consistent.

Reply:
OK, it's to get rid of the warning. Maybe add a comment?

Reply:
Sounds good, will do.